apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1/project.json

```diff
@@ -20,7 +20,7 @@
 
   "dependencies": {
     "Google.Api.CommonProtos": "1.0.0-beta02",
```

Comment:
There's some inconsistency with the Google.Api.CommonProtos version (beta02 vs beta04). Is this deliberate or should they all be the same?